### PR TITLE
Simplify information fragment layout (COMMUNITY)

### PR DESCRIPTION
**Pull request by fynngodau**
Using `ConstraintLayout` for the "App information" list makes the layout unnecessarily complicated because each row needs to reference the previous one. To avoid that, I recommend using a vertical `LinearLayout` like in this PR. It probably also increases performance by a minor bit.

The appearance of the layout should be the same:

appinformation

### CODE REVIEW

**Comment by d4rken** [comment]
Can you target branch `release/1.9.x` with your PR?

**Comment by fynngodau** [comment]
I don't think I can do that without opening a new PR, can I?

**Comment by fynngodau** [comment]
Oh, I can, I see. I rebased my branch onto `release/1.9.x` as well.

**Comment by d4rken** [comment]
Your PR currently contains unrelated changes.

**Comment by fynngodau** [comment]
Sorry, I didn't notice. I'm not sure how that happened. I removed the unrelated commit that I accidentally picked up somewhere.

File: Corona-Warn-App/src/main/res/layout/fragment_information.xml [hunk 1]
```diff
@@ -29,14 +29,16 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/information_header">
 
-            <androidx.constraintlayout.widget.ConstraintLayout
+            <LinearLayout
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content">
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/spacing_normal"
```
Comment (d4rken) [inline Corona-Warn-App/src/main/res/layout/fragment_information.xml:35]: Why did you add spacing here?

**Comment by d4rken** [comment]
Except for the new padding looks as intended, the padding isn't too bad either, up to you @harambasicluka

File: Corona-Warn-App/src/main/res/layout/fragment_information.xml (hunk 1 above)
Comment (fynngodau) [reply Corona-Warn-App/src/main/res/layout/fragment_information.xml:35]: I meant to move it from line 43 to here, and in the process duplicated it accidentally. I updated my PR to remove the other occurence.

**Review by harambasicluka — APPROVED** [review]
LGTM, thanks for your contribution :)
